mcbackend/adapters/pymc.py

```diff
@@ -159,12 +161,16 @@ def setup(
                     self._stat_groups.append([])
                     for statname, dtype in names_dtypes.items():
                         sname = f"sampler_{s}__{statname}"
-                        svar = Variable(
-                            name=sname,
-                            dtype=numpy.dtype(dtype).name,
-                            # This 👇 is needed until PyMC provides shapes ahead of time.
-                            undefined_ndim=True,
-                        )
+                        if statname == "warning":
```

Comment:
Meaning that if we would want to record warnings with hopsy we need to give the stat the name "warning"?

Reply:
Yes, for now this is only for stats named "warning", which also get special treatment by PyMC.

One could extend this to more arbitrary objects, but generally I think we should try to stick to standard data types and avoid objects, so I didn't want to add more flexibility than needed.